Tests/test_GenBank.py

```diff
@@ -1597,7 +1593,9 @@ def test_record_parser_13(self):
             record = next(records)
             length = 1755
             locus = "AB000050"
-            definition = "Feline panleukopenia virus DNA for capsid protein 2, complete cds"
```

Comment:
If this doesn't fit on the same line, how can line 1522 fit in one line?

Reply:
The old version had one line, length 92 (too long).

The new version has three lines, the middle one is 83 long.

So this does make sense - but I agree the example above is different.